liblll/CodeFragment.cpp

```diff
@@ -113,6 +113,23 @@ CodeFragment::CodeFragment(sp::utree const& _t, CompilerState& _s, bool _allowAS
 	}
 }
 
+namespace
+{
+/// Returns true iff the instruction is valid as a function. Assumes @a us is not nullptr.
+bool validFunctionalInstruction(string us)
+{
+	if (!c_instructions.count(us))
+		return false;
+	auto it = c_instructions.find(us);
+	return !(
```

Comment:
Alternatively `if (it == nullptr) return false;` could remove the need of `count()`?

Reply:
Yes. Precisely: `if (it == c_instructions.end())`. See http://en.cppreference.com/w/cpp/container/map/find "return value" section.

Reply:
Updated.